ansible/roles/idm-server/tasks/configure_idm.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: get local ip
+  shell: "ip addr show eth0 | grep 'inet ' | awk '{print $2}' | cut -d/ -f1"
+  register: private_ip
```

Comment:
You can use the ansible fact `ansible_default_ipv4.address`

Reply:
I believe we are not gathering facts in our playbooks, so would this fact be available?

Reply:
good point, you need [`setup`](https://docs.ansible.com/ansible/latest/modules/setup_module.html):

```yaml
- hosts: localhost
  gather_facts: no
  connection: local
  tasks:
          - setup:
              gather_subset:
                - network
          - debug:
                  msg: "{{ansible_default_ipv4.address}}"
```